config/jobs/kubernetes-sigs/kind/kind-presubmits.yaml

```diff
@@ -200,59 +200,4 @@ presubmits:
         hostPath:
           path: /sys/fs/cgroup
           type: Directory
-  # conformance test against kubernetes release-1.11 branch with `kind`, skipping
```

Comment:
why is this removed?

Reply:
note this is a sigs.k8s.io/kind presubmit, not a kubernetes/kubernetes presubmit

Reply:
I thought 1.11 wasn't supported any more. I've left it in in the next commit/push.

Reply:
it is for kubernetes proper, but kind is still tentatively supporting it :sweat_smile: 
I don't think we enforce sig subprojects to abandon 1.11?